common/src/main/cpp/classes/openxr_fb_passthrough_geometry.cpp

```diff
@@ -49,27 +59,20 @@ void OpenXRFbPassthroughGeometry::set_mesh(const Ref<Mesh> &p_mesh) {
 	mesh = p_mesh;
 
 	if (mesh.is_null()) {
-		if (preview_mesh != nullptr) {
-			preview_mesh->queue_free();
-			preview_mesh = nullptr;
+		if (opaque_mesh != nullptr) {
+			remove_child(opaque_mesh);
+			opaque_mesh->queue_free();
+			opaque_mesh = nullptr;
```

Comment:
This code block is repeated multiple times; can you wrap it into a `delete_opaque_mesh` helper function.